tests/templates/kuttl/resources/00-install-vector-aggregator-discovery-configmap.yaml.j2

```diff
@@ -0,0 +1,9 @@
+{% if lookup('env', 'VECTOR_AGGREGATOR') %}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: vector-aggregator-discovery
```

Comment:
The vector-aggregator-discovery ConfigMap must also be referenced in the cluster specs of all integration tests and the Vector agent must be enabled, e.g.:

```yaml
---
apiVersion: hive.stackable.tech/v1alpha1
kind: HiveCluster
metadata:
  name: test-hive
spec:
  image:
    productVersion: "{{ test_scenario['values']['hive'].split('-stackable')[0] }}"
    stackableVersion: "{{ test_scenario['values']['hive'].split('-stackable')[1] }}"
  clusterConfig:
{% if lookup('env', 'VECTOR_AGGREGATOR') %}
    vectorAggregatorConfigMapName: vector-aggregator-discovery
{% endif %}
  metastore:
    config:
      logging:
        enableVectorAgent: {{ lookup('env', 'VECTOR_AGGREGATOR') | length > 0 }}
    roleGroups:
      default:
        replicas: 1
```

Reply:
done https://github.com/stackabletech/hive-operator/pull/298/commits/f01b629d36a7587d7f21b8e352b9ce693733af34